td-shim-tools/Cargo.toml

```diff
@@ -23,8 +23,14 @@ required-features = ["signer"]
 name = "td-shim-checker"
 required-features = ["loader"]
 
+[[bin]]
+name = "td-reproducible-tool"
```

Comment:
Should this be renamed to "td-shim-xxxx"?

Reply:
I am OK with that.
Do you have a proposal? "td-shim-strip-info" ?